test/integration/util/util.go

```diff
@@ -132,7 +132,7 @@ func CreateResourceClaimController(ctx context.Context, tb ktesting.TB, clientSe
 	podInformer := informerFactory.Core().V1().Pods()
 	claimInformer := informerFactory.Resource().V1beta1().ResourceClaims()
 	claimTemplateInformer := informerFactory.Resource().V1beta1().ResourceClaimTemplates()
-	claimController, err := resourceclaim.NewController(klog.FromContext(ctx), true /* admin access */, clientSet, podInformer, claimInformer, claimTemplateInformer)
+	claimController, err := resourceclaim.NewController(klog.FromContext(ctx), true /* admin access */, true /* prioritized list */, clientSet, podInformer, claimInformer, claimTemplateInformer)
```

Comment:
This set of booleans is becoming a bit silly - sorry for even starting with it!

Should we add `1<<iota` enums with a custom int type such that this code can be come `resourceclaim.NewController(klog.FromContext(ctx), resourceclaim.DRAAdminAccessFeatureEnabled|resourceclaim.DRAPrioritizedListEnabled, ...`?

Not required, it's quite a bit of boilerplate code.

Reply:
We could do that. I already planned to look at this for the partitionable devices feature, since adding the third boolean here really made it awkward. But I'll take a look at it here.

Reply:
So I've made a change where I added a `Features` type that gets passed into the controller rather than the individual booleans. 
I also looked into using bitmasks. It is very convenient when the caller to the controller makes the decision on which features should be enabled (like in the example here), but gets a bit awkward when the caller have other variables that determines the features that should be enabled. Happy to look at it again if there are better ways to do it.